src/components/progress/progress.stories.tsx

```diff
@@ -19,17 +22,31 @@ type Story = StoryObj<typeof Progress>;
 /**
  * Default usage example of the Progress component.
  */
-export const Default: Story = {
+export const MD: Story = {
+    args: {
+        value: 10,
+        variant: 'md',
+    },
+};
+
+/**
+ * Thin usage example of the Progress component.
+ */
+export const SM: Story = {
     args: {
         value: 10,
+        variant: 'sm',
     },
 };
 
-export const Animation = () => {
+/**
+ * Separate functional component for animated progress.
+ */
+const AnimatedProgress: React.FC<IProgressProps> = ({ variant }) => {
```

Comment:
Same here, let's just pass all the props to the `<Progress />` component so that we won't need to update this story if we add more properties
```suggestion
const AnimatedProgress: React.FC<IProgressProps> = (props) => { 
```